integration_tests/src/main/python/array_test.py

```diff
@@ -394,3 +394,80 @@ def test_array_max_q1():
+        ('b', ArrayGen(data_gen, nullable=True))],
+        nullable=False)
+
+    assert_gpu_and_cpu_are_equal_collect(
+        lambda spark: gen_df(spark, gen).selectExpr(
+            'sort_array(array_intersect(a, b))',
+            'sort_array(array_intersect(b, a))',
+            'sort_array(array_intersect(a, array()))',
+            'sort_array(array_intersect(array(), b))',
+            'sort_array(array_intersect(a, a))',
+        )
+    )
+    
+@pytest.mark.parametrize('data_gen', [byte_gen, short_gen, int_gen, long_gen,
+    FloatGen(special_cases=[]), DoubleGen(special_cases=[]), string_gen, boolean_gen, date_gen, timestamp_gen], ids=idfn)
+def test_array_union(data_gen):
+    gen = StructGen(
+        [('a', ArrayGen(data_gen, nullable=True)),
+        ('b', ArrayGen(data_gen, nullable=True))],
+        nullable=False)
+
+    # The 4th item in this integration test here is left commented out here
+    # There is an issue with running that item with collect(), which affects
+    # the integration test here.
+    # See this issue (https://github.com/NVIDIA/spark-rapids/issues/5957) 
+    assert_gpu_and_cpu_are_equal_collect(
+        lambda spark: gen_df(spark, gen).selectExpr(
+            'sort_array(array_union(a, b))',
+            'sort_array(array_union(b, a))',
+            'sort_array(array_union(a, array()))',
+            # 'sort_array(array_union(array(), b))', # see https://github.com/NVIDIA/spark-rapids/issues/5957
+            'sort_array(array_union(a, a))',
+        )
+    )
+
+@pytest.mark.parametrize('data_gen', [byte_gen, short_gen, int_gen, long_gen,
+    FloatGen(special_cases=[]), DoubleGen(special_cases=[]), string_gen, boolean_gen, date_gen, timestamp_gen], ids=idfn)
+def test_array_except(data_gen):
+    gen = StructGen(
+        [('a', ArrayGen(data_gen, nullable=True)),
+        ('b', ArrayGen(data_gen, nullable=True))],
+        nullable=False)
+
+    assert_gpu_and_cpu_are_equal_collect(
+        lambda spark: gen_df(spark, gen).selectExpr(
+            'sort_array(array_except(a, b))',
+            'sort_array(array_except(b, a))',
+            'sort_array(array_except(a, array()))',
+            'sort_array(array_except(array(), b))',
+            'sort_array(array_except(a, a))',
+        )
+    )
+
+@pytest.mark.parametrize('data_gen', [byte_gen, short_gen, int_gen, long_gen,
+    FloatGen(special_cases=[]), DoubleGen(special_cases=[]), string_gen, boolean_gen, date_gen, timestamp_gen], ids=idfn)
+def test_arrays_overlap(data_gen):
+    gen = StructGen(
+        [('a', ArrayGen(data_gen, nullable=True)),
+        ('b', ArrayGen(data_gen, nullable=True))],
+        nullable=False)
+
+    assert_gpu_and_cpu_are_equal_collect(
+        lambda spark: gen_df(spark, gen).selectExpr(
+            'arrays_overlap(a, b)',
+            'arrays_overlap(b, a)',
+            'arrays_overlap(a, array())',
+            'arrays_overlap(array(), b)',
+            'arrays_overlap(a, a)',
+        )
+    )
```

Comment:
There should be an empty line at the end of the file as an implicit convention.
```suggestion
    )
    
```